webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,14 +7,56 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
-      <hc-teaser-image :contribution="contribution" @addTeaserImage="addTeaserImage">
+      <hc-teaser-image
+        :contribution="contribution"
+        @addTeaserImage="addTeaserImage"
+        :class="{ 'images-set-blur': checkedBlur }"
+      >
         <img
           v-if="contribution"
           class="contribution-image"
           :src="contribution.image | proxyApiUrl"
         />
       </hc-teaser-image>
+      <ds-text align="right" class="blurBox">
+        <div v-show="checkedBlur">
+          <ds-button
+            v-if="contribution"
+            class="bluricon-post"
+            icon="ban"
+            primary
+            @click.prevent="unBlur"
+          ></ds-button>
+          <img
+            v-if="contribution"
+            :src="contribution.image | proxyApiUrl"
+            class="blurImgPreview"
```

Comment:
From what I understand it's good to, `Use Hyphen Delimited Strings` for `CSS` names, no @alina-beck??
Here for reference https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/